pwiz_tools/Skyline/SkylineTester/SkylineTesterWindow.cs

```diff
@@ -1991,6 +1991,8 @@ private void runMode_SelectedIndexChanged(object sender, EventArgs e)
                 {
                     runLoops.Checked = true;
                     runLoopsCount.Text = 1.ToString();
+                    Offscreen.Checked = false;  // Can't do screenshots offscreen

```

Comment:
This doesn't appear to be re-enabled when the control is switched back to Test

Reply:
Probably want a behavior more like what you get when parallel mode is turned on/off

Reply:
Good catch. I think I fixed the potential inconsistencies the code was allowing before.